parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -310,41 +309,48 @@ removeContainerImage() {
     fi
 }
 
-cleanUpHyperkubeImages() {
-    echo $(date),$(hostname), startCleanUpHyperkubeImages
-    function cleanUpHyperkubeImagesRun() {
-        images_to_delete=$(docker images --format '{{OpenBraces}}.Repository{{CloseBraces}}:{{OpenBraces}}.Tag{{CloseBraces}}' | grep -vE "${KUBERNETES_VERSION}$|${KUBERNETES_VERSION}.[0-9]+$|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep 'hyperkube')
+cleanUpAllImages() {
```

Comment:
cleanUpImages?  It's actually cleaning any image, but not all

Reply:
renaming o cleanupImages

Reply:
still not resolved.

Reply:
nvm. Looked at the wrong iteration.